Tools/MiniBrowser/Scripts/process-minibrowser-entitlements.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+function plistbuddy()
+{
+    /usr/libexec/PlistBuddy -c "$*" "${WK_PROCESSED_XCENT_FILE}"
+}
+
+# ========================================
+# Mac entitlements
+# ========================================
+
+function mac_process_minibrowser_entitlements()
+{
+    # Start with the template entitlements from MiniBrowser.entitlements
+    plistbuddy Merge "${SCRIPT_INPUT_FILE_0}"
+}
+
+rm -f "${WK_PROCESSED_XCENT_FILE}"
+plistbuddy Clear dict
+
+if [[ "${WK_PLATFORM_NAME}" == macosx ]]
+then
+    if [[ "${PRODUCT_NAME}" == MiniBrowser ]]; then mac_process_minibrowser_entitlements
+    else echo "Unsupported/unknown product: ${PRODUCT_NAME}"
+    fi
+else
+    echo "Unsupported/unknown platform: ${WK_PLATFORM_NAME}"
+fi
```

Comment:
I know this is copied from other projects, but it _does_ feel like overkill, since MiniBrowser is Mac-only and one target.

Reply:
I left it in to shout if the script is unexpectedly called from a different build configuration.